upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -240,7 +240,7 @@ if [[ $state_recorded == "0" ]]; then
         [[ -n ${USERNAME} ]]
 
         # Install our pit-init RPM and pull in any dependencies it has.
-        zypper --no-gpg-checks in -y pit-init
+        zypper --no-gpg-checks --plus-repo=https://packages.local/repository/csm-sle-15sp2 in -y pit-init
```

Comment:
For 1.3, should we bake pit-init into the NCNs?  I know not all of the scripts are relevant in an NCN context, but here we are installing it anyway.